### DISCUSSION TITLE
New user issues/questions

### DISCUSSION THREAD

**Discussion by Ellwindrae** [oae/kaizoku#110] [Q&A]
Hello, 

Got Kaizoku set up and rolled into my Kavita setup, but running into a few things that aren't making since to me. The rest has been a pretty awesome experience. 

When I click on a series I see the records after I add the series, but if I go back to it it shows empty as below:
![image](https://github.com/oae/kaizoku/assets/139933416/22e596fc-9083-492c-b1cb-b1b2ff99794a)

I also get a handful of failed syncs with the following error: 
Error: Command failed with exit code 2: mangal inline --source Mangadex --query Witch Hat Atelier --manga exact --chapters 151 -d
panic: runtime error: index out of range [79] with length 79

Planning on going to the mangal github and asking there, but figured I'd ask here in case someone else has seen it. 

Thank you!

**Comment by oae** [comment]
Can you try removing the cache as mentioned [here](https://github.com/oae/kaizoku/issues/89#issuecomment-1549583824)
